Entities/Industry/Saw/Saw.as

```diff
@@ -238,6 +238,33 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid)
 		Vec2f bpos = blob.getPosition();
 		this.Tag("sawed");
 		this.server_Hit(blob, bpos, bpos - pos, 0.0f, Hitters::saw);
+
+		if (blob.getName() == "bomb")
+		{
+			if (this.get_u8("bombs_exploded") == 0)
```

Comment:
the `"bombs_exploded"` property is never set before. i assume that `get_u8` (also used by `add_u8` internally) assigns `0` as a default value but i suspect this causes a warning and should not be relied upon anyway.